commons/src/config/preamble.rs

```diff
@@ -54,208 +50,21 @@ pub struct I18n {
 
     #[arg(long, value_parser = parse_to_hashset::<Locale>, required = false, default_value = "")]
     #[serde(with = "locale::serde::multiple")]
-    pub langs: HashSet<Locale>,
-
-    #[arg(long = "locales-file")]
-    pub locales_file: Option<PathBuf>,
-
-    #[arg(long = "download-locales")]
-    pub download: bool,
+    pub output_langs: HashSet<Locale>,
```

Comment:
Why the change to `output_langs`? Does the flag used also change?

Reply:
this flag is now for output only. before, this flag had a kind of double use, because we also derived the locales that might appear in the document. now, it is only to indicate the languages that a document should be translated to. I will also update this in the spec accordingly